example/properties_directory/non-scalar/charge_density_profile.json

```diff
@@ -0,0 +1,196 @@
+{
+    "name": "charge_density_profile",
+    "xAxis": {
+        "label": "z coordinate"
+    },
+    "xDataArray": [
+        -4.89,
```

Comment:
I would suggest keeping list size to 3 maximum for brevity - we use ESSE in many places and should pay extra caution to that

Reply:
Done.